great_ape_safe/great_ape_safe.py

```diff
@@ -252,7 +252,9 @@ def post_safe_tx(self, skip_preview=False, events=True, call_trace=False, reset=
         # events, call_trace and reset are params passed to .preview
         # silent=True: prevent printing of safe_tx attributes at end of run
         # post=True: make the actual live posting of the tx to the gnosis api
-        if not safe_tx:
+        if not safe_tx and replace_nonce:
+            safe_tx = self.multisend_from_receipts(safe_nonce=replace_nonce)
```

Comment:
shouldn't this be handled in L262 by setting safe_tx._safe_nonce?

Reply:
This for an extra off case in which the transaction data can't be fetched either such as in this one.